src/components/TeacherViews/Onboarding/HowCanOakSupport/HowCanOakSupport.view.tsx

```diff
@@ -86,17 +81,28 @@ const HowCanOakSupport = () => {
         trigger={trigger as UseFormTrigger<OnboardingFormProps>}
         forceHideNewsletterSignUp={true}
         subheading="Select all that apply"
+        onSubmit={() => {
+          setValue("submitMode", "continue");
+          trigger();
+        }}
         secondaryButton={(isSubmitting) => (
           <OakSecondaryButton
             width="100%"
             disabled={hasMissingFormData || isSubmitting}
             name="skip"
+            onClick={() => {
+              setValue("submitMode", "skip");
+              trigger();
+            }}
```

Comment:
Flip `submitMode` between continue and skip to trigger the corresponding validation and transforms from the schema.

I'm not sure if this is idiomatic use of react-hook-form but the docs and discussions were unhelpful. I think another way would have been to fork the behaviour in the form `resolver` but that would still need someway to flip some state to say which button was clicked. I saw some examples that used an `onClick` on the secondary button to trigger `handleSubmit` but that would have required widespread refactoring of `OnboardingForm` which would have been a nightmare.